qiskit_ibm_runtime/visualization/draw_layer_error_map.py

```diff
@@ -39,7 +39,7 @@ def draw_layer_error_map(
     background_color: str = "white",
     radius: float = 0.25,
     width: int = 800,
-) -> go.Figure:
+) -> "PlotlyFigure":
```

Comment:
```suggestion
) -> PlotlyFigure:
```

Reply:
This is done on purpose so that the module is importable when plotly is not installed. The idea is that TYPE_CHECKING will be false at user import time.